src/Microsoft.ComponentDetection.Orchestrator/Orchestrator.cs

```diff
@@ -202,10 +202,10 @@ private async Task GenerateEnvironmentSpecificTelemetryAsync(BcdeExecutionTeleme
 
                         return string.Join(Environment.NewLine, aptListResult.Split(Environment.NewLine).Where(x => x.Contains("libssl")));
                     });
-
-                    if (!getLibSslPackages.Wait(taskTimeout))
+                    await getLibSslPackages.WaitAsync(taskTimeout);
+                    if (getLibSslPackages.IsFaulted)
```

Comment:
I am not sure if this is correct. I think `WaitAsync` will throw `TaskCanceledException`.

Reply:
REmovved the duplicate exception.